cylc/flow/cfgspec/workflow.py

```diff
@@ -1271,6 +1271,12 @@ def get_script_common_text(this: str, example: Optional[str] = None):
 
                     Task instances must be set to fail by
                     :cylc:conf:`[..]fail cycle points`.
+
+                    .. warning::
```

Comment:
(Warning is a bit much I think)
```suggestion
                    .. note::
```